examples/pushmi_pullyu.yml

```diff
@@ -17,6 +17,8 @@ process_name: pushmi_pullyu
 queue_name: dev:pmpy_queue
 minimum_age: 0
 rollbar_token: 'abc123xyz'
+rollbar_proxy_host: 'your_proxy_host_url'
```

Comment:
Wondering if it be better to name space rollbar now? like the rest?

```rollbar: 
  token: 'etc'
  proxy_host: 'etc'
  proxy_url: 'etc
```

Also might not hurt to write a test to make sure these are being set accordingly? (copy and paste the `rollbar_token` test if you want)